neuro_flow/batch_runner.py

```diff
@@ -167,7 +167,7 @@ async def process(
                     if st.id in finished:
                         continue
                     status = await self._client.jobs.status(st.raw_id)
-                    if status.status in (JobStatus.FAILED, JobStatus.SUCCEEDED):
+                    if status.status in (JobStatus.FAILED, JobStatus.SUCCEEDED, JobStatus.CANCELLED):
```

Comment:
I am wondering whether it is worth to add `JobStatus.is_finished()` or `JobStatus.FINISHED` (`= frozenset({JobStatus.FAILED, JobStatus.SUCCEEDED, JobStatus.CANCELLED})`)?

Reply:
You are completely right, I added TERMINATED_JOB_STATUSES and RUNNING_JOB_STATUSES sets (no-frozen) it in 01e240f, please check whether it is OK.